distribution/src/main/resources/bin/plugin.bat

```diff
@@ -9,9 +9,46 @@ for %%I in ("%SCRIPT_DIR%..") do set ES_HOME=%%~dpfI
 
 TITLE Elasticsearch Plugin Manager ${project.version}
 
+SET properties=
+SET args=
+
+:loop
+SET "current=%~1"
+SHIFT
+IF "x!current!" == "x" GOTO breakloop
+
+IF "!current:~0,2%!" == "-D" (
```

Comment:
> This means that properties can be passed as either -Dmy.prop true or -Dmy.prop=true

ha! I did not even know we can do this! thanks for clarifying. But that means that it is different in `elasticsearch.bat` and also the bash elasticsearch script. I just checked, `-Dmy.prop true` does not work with them. I will open an issue for it.